api/core/tools/provider/builtin/sparkimagegeneration/sparkimagegeneration.yaml

```diff
@@ -0,0 +1,62 @@
+identity:
+  author: Onelevenvy
+  name: sparkimagegeneration
```

Comment:
`sparkimagegeneration` is confusing, I would like it to be `spark`, the followings are my reasons
1. Spark is a model provider instead of image model generation provider, there will be more models to be released which can be used as tool, for example, image understanding, ppt generation.
2. provider name should be in `snake case` standard, join all words directly just make confusions to other developers.